documentation/commands/README.md

```diff
@@ -1,3 +1,475 @@
-# Cg commands
+# cg commands
 This is documentation for cg commands.
 
+## cg add
+Add new things to statusDB.
+
+### cg add customer
+Add a new customer with a unique INTERNAL_ID and NAME.
+
+```$ cg add customer CUSTOMER_ID CUSTOMER_NAME```
+
+where `CUSTOMER_ID` our internal customer ID, e.g. cust000, and `CUSTOMER_NAME` is the customer name.
+
+### cg add family
+Add a family to CUSTOMER_ID with a NAME.
+
+```$ cg add family CUSTOMER_ID FAMILY_NAME```
+
+where `FAMILY_NAME` is the customer's ID for the case.
+
+It is required to supply a panel.
+
+```$ cg add family --panel PANEL CUSTOMER_ID FAMILY_NAME```
+
+where `PANEL` are the gene panels.
+
+When adding a family, set the priority by using
+
+```$ cg add family --priority PRIORITY CUSTOMER_ID FAMILY_NAME```
+
+where `PRIORITY` could be `low`, `normal`, or `high`.
+
+### cg add relationship
+Create a link between a FAMILY_ID and a SAMPLE_ID.
+
+```$ cg add relationship FAMILY_ID SAMPLE_ID```
+
+where `FAMILY_ID` is our internal family ID, e.g. `cuddlygull`.
+
+Create a link between a FAMILY_ID and a SAMPLE_ID.
+
+```$ cg add relationship FAMILY_ID SAMPLE_ID```
+
+where `FAMILY_ID` is our internal family ID, e.g. `cuddlygull`.
+
+It is required to set the status.
+
+```cg add relationship --status STATUS FAMILY_ID SAMPLE_ID```
+
+where `STATUS` can be `affected`, `unaffected`, or `unknown`.
+
+Add relationship between parents using the `--mother` and `--father` flags. Example:
+
+```$ cg add relationship --mother MOTHER_ID FAMILY_ID SAMPLE_ID```
+
+where `MOTHER_ID` is the sampleID for the mother of the child sample.
+
+### cg add sample
+Add a sample for CUSTOMER_ID with a SAMPLE_NAME.
+
+```$ cg add sample CUSTOMER_ID SAMPLE_NAME```
+
+where `SAMPLE_NAME` is the customer's ID for the sample.
+
+It is required to set the gender of the sample.
+
+```$ cg add sample --sex SEX CUSTOMER_ID SAMPLE_NAME```
+
+where `SEX`can be `male`, `female`, or `unknown`.
+
+It is also required to set the application tag.
+
+```$ cg add sample --application APP_TAG CUSTOMER_ID SAMPLE_NAME```
+
+where `APP_TAG` is the application tag.
+
+Add a LIMS id by using the ´--lims´ flag.
+
+```$ cg add sample --lims LIMS_ID CUSTOMER_ID SAMPLE_NAME```
```

Comment:
explain what a lims id is.

We seem to be using sampleID for both external sampleID and limsID. To avoid confusing, it might be a good idea to add a table at the start of this document nailing down the nomenclature:
- sample_id = 'ADM..'
- ext_sample_id = 'customer sample_id'
- customer_id = custxxx
- ticket_id = 898988
- family_id = 'cuddlygull'

This way you also do not have to define all the above for each command that uses it